src/plzinit/init.go

```diff
@@ -1,14 +1,16 @@
 package plzinit
 
 import (
+	"bufio"
 	"fmt"
-	"gopkg.in/op/go-logging.v1"
 	"io/ioutil"
 	"os"
 	"path"
 	"path/filepath"
 	"strings"
 
+	"gopkg.in/op/go-logging.v1"
```

Comment:
Thanks :)

Reply:
Oops, that was unintentional. :)

Reply:
No no that's how we should be doing imports so it's good

Reply:
Yeah, but I don't like these changes sneaking into other changes. I prefer at least separate commits.